### DISCUSSION TITLE
Thanks !

### DISCUSSION THREAD

**Discussion by dburea01** [dedoc/scramble#354] [Show and tell]
Good morning
I just discovered "scramble", a bit by chance while trying to solve a problem with l5-swagger (darkaonline).

And what a shock! Barely installed, without touching the controllers or requests, I immediately obtained quality documentation.

Without any annotations that affect the readability of the code.
Here my first api doc lo learn how it works : https://notesapi-16ff4c1d6469.herokuapp.com/docs/api#/ 

Thank you for this to the whole team. And of course I'm going to talk about it around me.

**Comment by tasoskele** [comment]
Hello!
Could you explain to me how you commented under "comment" and "status", @/organizations/{organization}, please? I cannot find anything in the documentation about it and I would like to comment my enums, since they are not supported.
Thank you!

**Reply by dburea01** [reply]
hello
Just a simple php comment like that, in the resource file 

public function toArray(Request $request): array
    {
        return [
            /** @var string The id of the organization (uuid) */
            'id' => $this->id,
            'name' => $this->name,
            /** @var string|null A comment of the organization */
            'comment' => $this->comment,
            /** @var string The status of the organization 'ACTIVE' or 'INACTIVE' */
            'status' => $this->status,
            'created_at' => $this->created_at,
            'created_by' => $this->created_by,
            'updated_at' => $this->updated_at,
            'updated_by' => $this->updated_by,
        ];
    }

**Reply by dburea01** [reply]
This little project is public, here :[https://github.com/dburea01/notes-api]( https://github.com/dburea01/notes-api)

**Reply by tasoskele** [reply]
Thank you very much!
